modules/tfm/tfm/boards/common/crypto_keys.c

```diff
@@ -17,11 +16,8 @@
 #include <hw_unique_key.h>
 #include <identity_key.h>
 
-#define TFM_NS_PARTITION_ID (-1)
-
-#ifndef MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER
-#error "MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER must be selected in Mbed TLS config file"
-#endif
+#define NUMBER_OF_ELEMENTS_OF(x) sizeof(x)/sizeof(*x)
```

Comment:
Is the ARRAY_SIZE macro usable here instead?

Reply:
Oh wasn't aware of that macro and just used what was used in other TF-M platforms. changed it to `ARRAY_SIZE`